specification/mixedreality/resource-manager/readme.go.md

```diff
@@ -13,10 +13,20 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2021-01
   - tag: package-2020-05
   - tag: package-2019-02-preview
 ```
 
+### Tag: package-2021-01 and go
+
+These settings apply only when `--tag=package-2021-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-01-01/$(namespace)
+```
```

Comment:
Do we make a typo here? The tag name here is different from the one in `batch`

And also we cannot put this package in `preview` subdirectory, because all of the swaggers in this tag are stable
```suggestion
``` yaml $(tag) == 'package-2021-01' && $(go)
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-01-01/$(namespace)
```
```

Reply:
Thanks for catching the typo. I wasn't sure about the folder structure, does the new one work?